### PR TITLE
[WIP] Litmus book for application specific loadgen (jupyter-notebook)

### PR DESCRIPTION
**Pull request by vibhor995**
Signed-off-by: vibhor995 <vibhor.kumar@mayadata.io>

1) Litmus book for jupyter loadgen

@ksatchit @dargasudarshan @gprasath

### CODE REVIEW

**Comment by vibhor995** [comment]
**output of SOT and EOT lr**
```Name:         jupyter-loadgen
Namespace:
Labels:       <none>
Annotations:  kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"litmus.io/v1alpha1","kind":"LitmusResult","metadata":{"annotations":{},"name":"jupyter-loadgen","namespace":""},"spec":{"testMetadata":{...
API Version:  litmus.io/v1alpha1
Kind:         LitmusResult
Metadata:
  Cluster Name:
  Creation Timestamp:  2018-09-10T10:55:09Z
  Generation:          0
  Resource Version:    3200050
  Self Link:           /apis/litmus.io/v1alpha1/jupyter-loadgen
  UID:                 fc532269-b4e7-11e8-8143-42010a800193
Spec:
  Test Metadata:
    App:        <nil>
    Chaostype:  <nil>
  Test Status:
    Phase:   completed
    Result:  Pass
Events:      <none>

**Comment by vibhor995** [comment]
**logs**  : i will be posting this in more verbose
```kubectl logs -f litmus-jupyter-loadgen-8f8pk -n litmus
No config file found; using defaults
 [WARNING]: Ignoring invalid attribute: executable

PLAY [localhost] ***************************************************************

TASK [Generate the litmus result CR to reflect SOT (Start of Test)] ************

TASK [Apply the litmus result CR] **********************************************

TASK [Checking the status  of test specific namespace.] ************************

TASK [Get the name of jupyter-notebook application pod.] ***********************

TASK [Copy and trigger the loadgen file inside jupyter application pod] ********
FAILED - RETRYING: Verify load generated. (5 retries left).

TASK [Verify load generated.] **************************************************

TASK [Generate the litmus result CR to reflect EOT (End of Test)] **************

TASK [Apply the litmus result CR] **********************************************

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=11   changed=8    unreachable=0    failed=0

**Comment by vibhor995** [comment]
```Every 2.0s: kubectl get po -n litmus                                              cs-6000-devshell-vm-75a6c500-8fb4-439d-90b1-dd2a0f489662: Mon Sep 10 20:48:43 2018

NAME                           READY     STATUS    RESTARTS   AGE
jupyter-liveness-gt2tc         1/1       Running   0          6h
litmus-jupyter-loadgen-76747   1/1       Running   0          17s

File: apps/jupyter/workload/test.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - name: Generate the litmus result CR to reflect SOT (Start of Test)
```
Comment (dargasudarshan) [inline apps/jupyter/workload/test.yml]: @vibhor995 Can you include common utils for app_deploy, deploy_check, litmus_results in this litmus book? where ever applicable?

Comment (vibhor995) [reply apps/jupyter/workload/test.yml]: on it @dargasudarshan

**Comment by ksatchit** [comment]
Looking at a non `kubectl exec` based approach for loadgen. Closing this for now, as this needs use of separate tooling.

**Pull request closed without merging**